vars/skipStage.groovy

```diff
@@ -213,8 +213,7 @@ boolean call(Map config = [:]) {
             return paramsValue('CI_BUILD_PACKAGES_ONLY', false) ||
                    skip_stage_pragma('build-centos7-gcc', 'false') ||
                    (docOnlyChange(target_branch) &&
-                    prRepos('centos7') == '') ||
-                   quickBuild()
+                    prRepos('centos7') == '')
```

Comment:
```suggestion
                    prRepos('centos7') == '') ||
                    quickFunctional()
```

Is likely what was originally intended.  QB runs UT but QF skips UT and therefore non-RPM builds are not necessary.

And yes, this entire stage-skipping code needs a redesign.

Reply:
I don't think I put that much effort into what was intended.  Whilst we're here what would be required to add checkpatch (and presumably pylint) back into quickFunctional()

Reply:
We've already had this discussion.  `Quick-functional`'s entire point of existence is to get to functional testing as quickly as possible.  Do not pass go, do not collect $100.  If you want functional only testing but with linting and checkpatch and whatnot (i.e. a functional-only testing path to landing), that is a new pragma.  Quick-functional's purpose is to iterate on functional testing as quickly as possible.  It's mean for development only.

Reply:
Yeah, and I got caught by it again last week where I waited 48 hours for a test-run only to find out I'd got the syntax wrong.